tests/integration/testutil/common.go

```diff
@@ -155,36 +158,38 @@ func CreateTestInput(t *testing.T) TestInput {
 		rewardtypes.ModuleName:         nil,
 	}
 
+	govModAddress := authtypes.NewModuleAddress(govtypes.ModuleName).String()
+
 	paramsKeeper := paramskeeper.NewKeeper(appCodec, legacyAmino, keyParams, tKeyParams)
-	accountKeeper := authkeeper.NewAccountKeeper(appCodec, keyAcc, paramsKeeper.Subspace(authtypes.ModuleName), authtypes.ProtoBaseAccount, maccPerms)
-	bankKeeper := bankkeeper.NewBaseKeeper(appCodec, keyBank, accountKeeper, paramsKeeper.Subspace(banktypes.ModuleName), blackListAddrs)
+	accountKeeper := authkeeper.NewAccountKeeper(appCodec, keyAcc, authtypes.ProtoBaseAccount, maccPerms, sdk.GetConfig().GetBech32AccountAddrPrefix(), govModAddress)
```

Comment:
There is no problem now, but I think it would be good to change it to `etherminttypes.ProtoBaseAccount` for later.